core/src/main/java/hudson/model/TopLevelItemDescriptor.java

```diff
@@ -114,6 +127,64 @@ public String getDisplayName() {
     }
 
     /**
+     * A description of this kind of item type. This description can contain HTML code but it is recommend to use text plain
+     * in order to avoid how it should be represented.
+     *
+     * @return A string, an empty string by default.
+     *
+     * @since TODO
+     */
+    @Nonnull
+    public String getDescription() {
+        try {
+            WebApp webapp = WebApp.getCurrent();
+            MetaClass meta = webapp.getMetaClass(this);
+            Script s = meta.loadTearOff(JellyClassTearOff.class).findScript("newInstanceDetail");
+            if (s == null) {
+                return "";
+            }
+            DefaultScriptInvoker dsi = new DefaultScriptInvoker();
+            StringWriter sw = new StringWriter();
+            XMLOutput xml = dsi.createXMLOutput(sw, true);
+            dsi.invokeScript(Stapler.getCurrentRequest(), Stapler.getCurrentResponse(), s, this, xml);
+            return sw.toString();
+        } catch (Exception e) {
+            return "";
+        }
+    }
+
+    /**
+     * Used to categorize this kind of item type. @see {@link ItemCategory}
+     *
+     * @return A string with the category identifier, {@link ItemCategory.UncategorizedCategory#getId()} by default.
+     *
+     * @since TODO
+     */
+    @Nonnull
+    public String getCategoryId() {
+        return new ItemCategory.UncategorizedCategory().getId();
+    }
+
+    /**
+     * @since TODO
```

Comment:
Yeah it seems a recent update to GitHub changes the display of PR diffs as hyperlinked from mail notifications, at least for comments now hidden by subsequent commits: it shows an older version of the diff, perhaps as it was when the original comment in the thread was made, or perhaps as it was just before the thread was hidden, I am not sure. You now need to click on **View all changes** to see the current diff.